spec/dummy/config/application.rb

```diff
@@ -32,6 +32,9 @@ class Application < Rails::Application
     # Ensure the tests only pass when all the required tralations are present.
     config.action_view.raise_on_missing_translations = true
 
+    # Change authomatic expire of renew's magic link token
+    config.renew_token_expires_in_days = ENV["RENEW_TOKEN_EXPIRES_IN_DAYS"] || 60
```

Comment:
Is 60 days right? I don't think this ticket specifies the token expiry, but RUBY-496 says the grace period ends one month after the original registration expires. If we go with days then we could have someone receive the email on 1 July, expire on 1 August and then be unable to renew on 29 August – so they'd be missing out on 2 days.

If we do decide to go with days I think we need to make sure everyone is clear on this so we are telling the user the right information. (eg "this will expire in 60 days" instead of "this will expire in 2 months").

Reply:
Good point. I'll ask about this in tomorrow's standup :)

Reply:
Whilst I think about it, it might be prudent to split the config into 2 as well so we have a bit of flexibility and it's clearer in the code what's going on.

So we'd have something like

- `REGISTRATION_RENEWAL_WINDOW=1`
- `REGISTRATION_GRACE_WINDOW=1`

The JWT token expiry would then be the sum of both values

> This idea is copied from Waste Carriers which splits the config

Reply:
Ooh, I agree with this ☝️

Reply:
Me too! But can I suggest we use the existing `FIRST_RENEWAL_EMAIL_REMINDER_DAYS`? Or, even better, `x days until expires` + `REGISTRATION_GRACE_WINDOW`?

Reply:
If I understood it correctly, what we want in reality is a token that is "active" from the moment it is generated until the last possible day before the renewal + a grace period, so let's code that :)

Reply:
@cintamani Sounds good - `FIRST_RENEWAL_EMAIL_REMINDER_DAYS + REGISTRATION_GRACE_WINDOW` makes sense to me! 👍